WinUIGallery/Helper/WindowHelper.cs

```diff
@@ -65,8 +67,26 @@ static public Window GetWindowForElement(UIElement element)
             return null;
         }
 
-        static public List<Window> ActiveWindows { get { return _activeWindows; }}
+        static public List<Window> ActiveWindows { get { return _activeWindows; } }
 
         static private List<Window> _activeWindows = new List<Window>();
+
+        public static bool IsAppPackaged
+        {
+            get
+            {
+                int bufferSize = 0;
+                byte byteBuffer = 0;
+                // Try to grab the ID of the Package of the current process (if it exists)
+                uint lastError = NativeHelper.GetCurrentPackageId(ref bufferSize, out byteBuffer);
+                bool isPacakged = true;
+
+                if (lastError == NativeHelper.APPMODEL_ERROR_NO_PACKAGE)
```

Comment:
What if you are unlucky and the call to GetCurrentPackageId() failed due to transient errors, such as "out of memory" or else? Currently you default to isPackage=true in that situation, can it cause problems downstream?

Reply:
Looking at the implementation of GetCurrentPackageId, the first check it makes is whether the app is packaged, short-circuiting out with APPMODEL_ERROR_NO_PACKAGE if not.  This approach appears safe.